release-notes/release-notes5014.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: TinyMCE 5.0.14
+title_nav: TinyMCE 5.0.14
+keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissues
+---
+
+## Overview
+
+These Release Notes provide a high-level coverage of: improvements and additions, known issues, important bug fixes & deprecated functionality associated with the release of TinyMCE 5.0.14.
+
+## New features
+
+### TinyMCE
+
+#### Autocompleter API
+
+The autocompleter API now provides a new `reload` function to allow the autocompleter menu to reload, with potentially more or different results. This could be used for a "More..." menu item that would load more results.
+
+See the autocompleter documentation for information: [https://www.tiny.cloud/docs/ui-components/autocompleter/](https://www.tiny.cloud/docs/ui-components/autocompleter/)
+
+
+### Accessibility Checker 2.1.0
+
+Two new settings have been introduced to allow developers to set the Accessibility Level and HTML version they wish to comply to. The new settings are:
+
+* `a11ychecker_level`  - a, aa or aaa (aa by default)
+* `a11ychecker_html_version` - html4 or html5 (html4 by default)
+
+Accessibility Checker 2.1.0 provides a new streamlined design for the accessibility checker dialog:
+
+![Redesigned Accessibility Checker Dialog]({{site.baseurl}}/images/accessibility_checker.png)
+
+> **Note:** We don't currently support any additional accessibility rules, but it does enable variations of the current rules to be checked. For example, enabling `aaa` will check the color contrast is 7:1 instead of 4.5:1.
+
+## Updates and enhancements
+
+### PowerPaste 5.0.2
+
+PowerPaste 5.0.2 now provides a way for developers to keep unsupported image source URLs in the HTML, instead of PowerPaste completely removing them. To enable set the new `powerpaste_keep_unsupported_src` setting to `true`, which will store the original `src` in the `data-image-src` attribute. This allows developers to replace the images manually using post-processing. For example, if the src is a `file://` URL and they have access to the local filesystem, they can add the image during post-processing of the posted content.
```

Comment:
`s/To enable set/To enable, set/`?

Reply:
Also we probably need to remove "they" in the last sentence and replace it with something like a reference to the developer?

Reply:
replacing with 'you'

Reply:
I thought we were supposed to avoid referring to someone directly (ie try to write in the third person)?

Reply:
fair. reworded.